engine/commands/cortex_upd_cmd.cc

```diff
@@ -153,8 +226,7 @@ bool CortexUpdCmd::GetNightly(const std::string& v) {
   CTL_INF("Engine release path: " << url_parser::FromUrl(url_obj));
 
   std::filesystem::path localPath =
-      file_manager_utils::GetExecutableFolderContainerPath() / "cortex" /
-      path_list.back();
+      std::filesystem::temp_directory_path() / "cortex" / path_list.back();
```

Comment:
I tested, and we will need to create the corresponding directory, or we will have a crash.

```
  try {
    std::filesystem::create_directories(localPath.parent_path());
  } catch (const std::filesystem::filesystem_error& e) {
    CTL_ERR("Failed to create directories: " << e.what());
    throw;
  }
```

Reply:
Thanks. Added `create_directories` if it does not exist.